Plugins/DeleteUnusedBranches/DeleteUnusedBranchesForm.cs

```diff
@@ -97,7 +97,7 @@ private IEnumerable<Branch> GetObsoleteBranches(RefreshContext context, string c
 
                 GitArgumentBuilder args = new("log")
                 {
-                    "--pretty=%ci\n%an\n%s",
+                    @"--pretty=""format:%ci\n%an\n%s""",
```

Comment:
This results in
`"--pretty=\"format:%ci\\n%an\\n%s\""`
in contrast to
`         "--pretty=%ci\n%an\n%s"`
before.
That's why the table is not filled.